src/redux/actions/BerthActions.tsx

```diff
@@ -0,0 +1,19 @@
+import { createAction } from 'redux-actions';
+
+const selectBerth = createAction('SELECT_BERTH', (id: string) => id);
+const deselectBerth = createAction('DESELECT_BERTH', (id: string) => id);
+const moveUp = createAction('MOVE_BERTH_UP', (id: string) => id);
+const moveDown = createAction('MOVE_BERTH_DOWN', (id: string) => id);
```

Comment:
The `id` type of the previous four actions should be `Berth` instead of `string`.

Reply:
Oh my mistake when i fix conflicts. Fixed